### ISSUE TITLE
Unable to resolve non-existing file error with Virtual File System

### ISSUE THREAD

**Issue by carlocardella** [DavidAnson/vscode-markdownlint#192]
The extension updated to `v0.44.0` this morning (or better, I restarted VSCode to load the new extension version) and now I get a number of errors as shown below. 
_Note_: this is a virtual document opened through [GistPad](https://marketplace.visualstudio.com/items?itemName=vsls-contrib.gistfs) (vsls-contrib.gistfs), it used to work fine with older releases.
Technically the error is correct, there is no such file on my D:\ drive (and it is not supposed to be there).

```
[8:42:28 AM] ERROR: Exception while linting with markdownlint-cli2:
EntryNotFound (FileSystemError): Unable to resolve non-existing file 'd:\journal\921.md'
    at _handleError (c:\Users\carloc\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\workbench\services\extensions\node\extensionHostProcess.js:94:160087)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

VSCode and OS Version
```
Version: 1.60.1 (user setup)
Commit: 83bd43bc519d15e50c4272c6cf5c1479df196a4d
Date: 2021-09-10T17:07:10.714Z
Electron: 13.1.8
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043
```

**Comment by DavidAnson** [comment]
I don't use this extension, but I'll try tonight.

**Comment by DavidAnson** [comment]
Sorry for the trouble! I've just published version 0.44.1 which is identical to the previous version 0.43.2. I will investigate a couple of bug reports and re-release the new version with fixes soon.

**Comment by carlocardella** [comment]
No problem, thanks for the update 👍
